docs/hypernode-platform/nginx/how-to-set-up-an-application-proxy-nginx.md

```diff
@@ -0,0 +1,50 @@
+---
+myst:
+  html_meta:
+    description: Learn how to set up an Nginx application proxy on Hypernode using Managed Vhosts: create a proxy vhost and point it to your app
+    title: How to set up an application proxy in Nginx? | Hypernode
+---
+
+# How to Set Up an Application Proxy in Nginx
+
+Sometimes you want to serve an application that listens on a local port (for example a Node, Python or PHP service on `localhost:3000`) behind a friendly domain on your Hypernode. Instead of writing a custom Nginx config, you can use Hypernode Managed Vhosts to generate a ready‑made proxy vhost coniguration and then just point it to your application. This keeps your configuration simple and consistent with the rest of your setup.
+
+## Create a proxy vhost
+
+Create a new vhost and set its type to `proxy`. In the example below we create a vhost for `proxy.myapp.hypernode.io`:
+
+```bash
```

Comment:
Missing some inconsistency with other docs:
- In these examples, use console over bash (because there's a prompt)
- Use `example` instead of `myapp`